.github/workflows/contract-testing.yaml

```diff
@@ -57,6 +59,10 @@ jobs:
           - 'cfn-resources/serverless-private-endpoint/**'
         cloud-backup-restore-jobs:
           - 'cfn-resources/cloud-backup-restore-jobs/**'
+        private-endpoint:
+          - 'cfn-resources/private-endpoint/**'
+        private-endpoint-service:
+          - 'cfn-resources/private-endpoint-service/**'
```

Comment:
was this split asked by a customer? Why did we decide to split the resource?

Reply:
Originally this resource was ment to be be split, private endpoint, and private endpoint service were two separated resources (since in terraform and on atlas they are devided), also this resource controlls an AWS private endpoint, that is quite comples, so now is way easier to maintain because we dont have to worry about the AWS private endpoint

Reply:
I am still not sure why we are making this change now. Customers using the resource will be confused by this change. Was this requested by @Zuhairahmed? What happens to customers currently using the resource and upgrading to the latest ones? 

We already have many resources (more than 30) and adding a new one should be done carefully.

Reply:
yes @andreaangiolillo we synced on this to better align with Terraform/other tools functionality. @Adelmar92 to be clear we are keeping existing Private Endpoint resource as is and just adding new AWS Private Endpoint Resource and new Private Endpoint Service Resource right?

Reply:
looking at below this appears to be the case, there should be no immediate change to quick start templates or breaking changes with this. also agree we will need new L1 constructors to support these 2 new resources